### PR TITLE
Ports the Baystation Journalism Camera! Part 2!

### PR DESCRIPTION
**Pull request by LonelyHowls**
Heya! This is me redoing https://github.com/Aurorastation/Aurora.3/pull/14528 so that it actually works and isn't pulling from my master. I also updated the map file. I was originally going to do this all through the first PR but, uh, I screwed it up and had to refork. Lmao.

Copy of the original text here: 

Hello! This PR ports the camera drone that is available on several other servers to Aurora. I'm not an especially great coder, however, and a lot of this work was handled by reverse engineering other things. I'm not totally sure if this is up to standard, but I'll make adjustments as necessary!

Things this PR does:

    Ports the journalist camera from Baystation and adds a single instance of the object in the journalist's locker. The drone can broadcast video to a channel of the user's naming and broadcasts audio to the entertainment network. This device can also be built by the machinist.
    Creates a News network. The network is currently only used for this. If that ends up being undesirable, I can also use the Thunderdome network since it's otherwise unused.
    Allows any user to download the camera application to their modular computer of choice. This is necessary to properly implement the camera, otherwise nobody would be able to see it! By default, the only accessible camera network without an ID will be the News network.

I don't know if I did anything wrong with this - please let me know if I did! I'd like to learn how to properly do this sort of thing. I have only done, like, maybe two or three PRs in the whole of my time playing SS13. ^^;

### CODE REVIEW

**Comment by LonelyHowls** [comment]
!review

**Comment by LonelyHowls** [comment]
I also addressed Matt's original reviews in this PR! Weee.

File: code/game/machinery/camera/presets.dm [hunk 1]
```diff
@@ -77,6 +77,10 @@ var/global/list/engineering_networks = list(
 /obj/machinery/camera/network/thunder
 	network = list(NETWORK_THUNDER)
 
+/obj/machinery/camera/network/news
+	network = list(NETWORK_NEWS)
+
+
```
Comment (NonQueueingMatt) [inline code/game/machinery/camera/presets.dm:83]:
```suggestion
```

File: code/game/objects/items/devices/tvcamera.dm [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+		icon_state = "camcorder"
+		item_state = "camcorder"
+	var/mob/living/carbon/human/H = loc
+	if(istype(H))
+		H.update_inv_r_hand(0)
+		H.update_inv_l_hand()
+
+/* Assembly by a roboticist */
+/obj/item/robot_parts/head/attackby(var/obj/item/device/assembly/S, mob/user as mob)
+	if ((!istype(S, /obj/item/device/assembly/infra)))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/devices/tvcamera.dm:85]:
```suggestion
	if (!istype(S, /obj/item/device/assembly/infra))
```

**Review by NonQueueingMatt — APPROVED** [review]
Minor fixes, but otherwise looks good to me.

**Comment by NonQueueingMatt** [comment]
Abandoned. DM me if you need this reopened.

**Pull request closed without merging**